### PR TITLE
Edits and a new translatable string

### PR DESCRIPTION
**Pull request by nataliechan**
I made a new translatable string so I could include info on how to change your email address, as mentioned by @Alys in #4468, but ran into issues when I tried to run grunt in order to see if the string I added worked, so feedback would be helpful, especially if it's obvious where I'm going wrong. (Also the second commit is the correct one, I accidentally copy-pasted something incorrectly in the first file).

### CODE REVIEW

**Comment by Alys** [comment]
When you have problems with compiling, or any software problems (for any project), it's always worth giving a full description of the problem, including an exact copy of all error messages and any related output. There could be many things causing problems with grunt so we do need to see exactly what's happening. 

Could you please adjust the final line of the file in this PR so that there's a line break before `}`, as there was before? That's useful to avoid future bugs (people might try adding new lines after `}` if it's hidden at the end of an existing line) and it maintains consistency with the other files.

**Comment by Alys** [comment]
Goes with https://github.com/HabitRPG/habitrpg/pull/4504

File: locales/en/settings.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "enterNumber": "Please enter a number between 0 and 24",
     "fillAll": "Please fill out all fields",
     "passSuccess": "Password successfully changed",
-    "usernameSuccess": "Username successfully changed",
+    "usernameSuccess": "Login Name successfully changed",
     "data": "Data",
-    "exportData": "Export Data"
-}
+    "exportData": "Export Data",
+    "emailChange": "To change your email address, please send an email to admin\@habitrpg.com including both your old and new email address as well as your User ID."}
```
Comment (Alys) [inline locales/en/settings.json]: If you could make the email address a mailto: link, that would be very helpful for the users! See how other `a href` links are done in this file.

**Comment by nataliechan** [comment]
Okidokies. I've described what I've done with grunt and npm start below. Please let me know if I'm still missing info that might be important to help with debugging. Sorry I'm still new at this and still trying to figure everything out!

So when I run <code>grunt</grunt> from node_modules/habitrpg_shared, it gives me the following:

<code>
natalie@ubuntu:~/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared$ grunt

> > Local Npm module "grunt-contrib-clean" not found. Is it installed?
> > Local Npm module "grunt-spritesmith" not found. Is it installed?
> > Local Npm module "grunt-contrib-cssmin" not found. Is it installed?
> > Local Npm module "grunt-browserify" not found. Is it installed?
> > Warning: Task "cssmin" not found. Use --force to continue.

Aborted due to warnings.
</code>

So after that I ran <code> npm install grunt-contrib-clean</code>, and the same for grunt-spritesmith, -contrib-cssmin, -browserify, and tried to run <code>grunt</code> again, and ran into:

<code>
Running "browserify:dist" (browserify) task

> > Error: Cannot find module 'coffeeify' from '/home/natalie/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared'
> > Warning: Error running grunt-browserify. Use --force to continue.

Aborted due to warnings.
</code>

Installed <coffeeify>, still ran into the same warning, used <code>--force</code> just to see what <code>npm start</code> would give, and got the following, multiple times:

<code>
module.js:485
    throw err;
          ^
SyntaxError: /home/natalie/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared/locales/en/settings.json: Unexpected token @
    at Object.parse (native)
    at Object.Module._extensions..json (module.js:482:27)
    at Module.load (/home/natalie/Documents/habitrpg/habitrpg/node_modules/coffee-script/lib/coffee-script/coffee-script.js:211:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /home/natalie/Documents/habitrpg/habitrpg/src/i18n.js:15:35
    at Function.forEach (/home/natalie/Documents/habitrpg/habitrpg/node_modules/lodash/dist/lodash.js:3297:15)
    at loadTranslations (/home/natalie/Documents/habitrpg/habitrpg/src/i18n.js:13:5)
    at Object.<anonymous> (/home/natalie/Documents/habitrpg/habitrpg/src/i18n.js:20:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (/home/natalie/Documents/habitrpg/habitrpg/node_modules/coffee-script/lib/coffee-script/coffee-script.js:211:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/natalie/Documents/habitrpg/habitrpg/src/server.js:31:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
info: [Sun Jan 11 2015 10:52:07 GMT-0500 (EST)] [master:18203] worker:18256 disconnect! new worker:18258 fork
</code>

**Comment by Alys** [comment]
Under ~/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared', run `npm install coffeeify` then if you still get coffeeify errors, under  ~/Documents/habitrpg/habitrpg/ run `find . -name \*coffeeify\*` 

`Unexpected token @` might be referring to `\@` in the email address. I don't believe the \ is necessary and it could be causing it to be interpreted as a special symbol. Might be unrelated but that's the first thing I'd change to try to fix this.

As a general rule FYI (not something you've done wrong!) I'm always very reluctant to use force modes (at least in the long term) because errors are often best fixed rather than avoided. However force during debugging can give useful information, as now. You're right to use it while troubleshooting, but I'd always want to work out how to make it unnecessary for production work. The install and find commands above should help us with that.

**Comment by nataliechan** [comment]
Back again. Still getting the coffeeify error (see below for output), but I managed to get things running I think. Screenshot below the output.

<code>
natalie@ubuntu:~/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared$ npm install coffeeify
npm WARN package.json habitrpg-shared@0.0.0 No repository field.
coffeeify@0.6.0 node_modules/coffeeify
├── convert-source-map@0.3.5
├── through@2.3.6
└── coffee-script@1.7.1 (mkdirp@0.3.5)
natalie@ubuntu:~/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared$ gruntRunning "cssmin:dist" (cssmin) task

Running "browserify:dist" (browserify) task

> > Error: Cannot find module 'coffeeify' from '/home/natalie/Documents/habitrpg/habitrpg/node_modules/moment'
> > Warning: Error running grunt-browserify. Use --force to continue.

Aborted due to warnings.
natalie@ubuntu:~/Documents/habitrpg/habitrpg/node_modules/habitrpg-shared$ find . -name *coffeeify*
./node_modules/grunt-browserify/node_modules/browserify/test/global_coffeeify.js
./node_modules/grunt-browserify/node_modules/browserify/test/coffeeify
./node_modules/grunt-browserify/node_modules/browserify/test/coffeeify.js
./node_modules/grunt-browserify/node_modules/watchify/node_modules/browserify/test/global_coffeeify.js
./node_modules/grunt-browserify/node_modules/watchify/node_modules/browserify/test/coffeeify
./node_modules/grunt-browserify/node_modules/watchify/node_modules/browserify/test/coffeeify.js
./node_modules/coffeeify
</code>

image

**Comment by Alys** [comment]
Also run `npm install coffeeify` under ~/Documents/habitrpg/habitrpg/ . Does that get rid of the error?

**Comment by deilann** [comment]
Actually, did you `npm install` in the shared repo at all?

**Comment by nataliechan** [comment]
Ahem. Yes. I've now run <code>npm install</code> and everything is green and runs without errors. Thanks Alys! I'll try to get the other stuff you suggested in the issue implemented soon. Should I comment here when the pull request is actually ready to go in, or should I make a new one?

**Comment by crookedneighbor** [comment]
@scribblypanda Commenting here is great.

**Pull request merged**